EasyPost/Models/API/Options.cs

```diff
@@ -133,6 +135,8 @@ public class Options : EasyPostObject
         public bool? PeelAndReturn { get; set; }
         [JsonProperty("pickup_min_datetime")]
         public DateTime? PickupMinDatetime { get; set; }
+        [JsonProperty("pickup_max_datetime")]
+        public DateTime? PickupMaxDatetime { get; set; }
```

Comment:
```suggestion
        [JsonProperty("pickup_max_datetime")]
        public DateTime? PickupMaxDatetime { get; set; }
        [JsonProperty("pickup_min_datetime")]
        public DateTime? PickupMinDatetime { get; set; }
```

Should be sorted order